src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -826,11 +826,7 @@
   {{ form_widget(form['max_field'], { attr: {class: 'max-field'}}) }}
 {% endblock %}
 
-{% block form_help %}
-  {% if help %}
-    <small class="form-text">{{ help }}</small>
-  {% endif %}
-{% endblock form_help %}
+{% block form_help %}{% endblock form_help %}
```

Comment:
Not sure the problem comes from the bootstrap layout :thinking:

Reply:
I'm not a twig specialist with extends templates in cascad but the block form_help is already defined in the prestashop_ui_kit_base.html.twig

https://github.com/PrestaShop/PrestaShop/blob/acaad10ffbf2c371b0f111aaa0702a4502328af1/src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig#L1122

Each specific block add `{{- block('form_help') -}}` to show help at the right place.

Reply:
Sorry for late answer.

bootstrap_4_layout.html.twig is the Original PrestaShop Form theme (see https://devdocs.prestashop.com/1.7/development/components/form/form-theme/form-theme/#prestashop-form-themes)

It's used to display forms in a vertical way.

prestashop_ui_kit_base.html.twig is the PrestaShop UI Kit Form theme we built in 1.7.7 to replace the old Form Theme.


It's used to display forms in a vertical way, just like previous one.

So these two form themes are not supposed to be mixed 🤔 I think this is why @PierreRambaud was unsure

Reply:
I will improve the doc https://github.com/PrestaShop/docs/pull/1375/

Reply:
@matks The checkform module use 
{% extends '@PrestaShop/Admin/layout.html.twig' %}

I understand there is an heritage of theme issue, but what’s the best to fix this issue ? The module need to extend an other template or native PrestaShop template has a bug ?